device/ospfv3/interface/auth.yaml

```diff
@@ -0,0 +1,70 @@
+components:
+  schemas:
+    Ospfv3Interface.Authentication:
+      description: |-
+        This contains OSPFv3 authentication properties.
+      type: object
+      properties:
+        enable:
```

Comment:
probably we dont need explicit "enable" configuring the object it is present or otherwise not.